### PR TITLE
Revamp/login page

### LINKED ISSUE #326: Redesign Login Page
## Expected Behavior

MedTagger should use new look and feel on the login page.

## Tasks to do

- [x] Design on Figma,
- [x] Implementation - CSS/HTML/JS.

### PR DESCRIPTION
**Pull request by juliapacula**
Fixes #326 

## Proposed Changes

  - new login-page view - compatible with design,
  - made a fluid layout so on smaller devices page is still readable
  - added bootstrap to styles and used grid layout
  - added logo to assets
  - the user is now able to see a password on login and register
  - login page starts now with sign up form
image
image
Tablet:
image
image
Iphone:
image
image

### CODE REVIEW

**Comment by jpowie01** [comment]
Please add some screenshots to the Pull Request message :)

**Comment by juliapacula** [comment]
Done :+1: Forgot about Lorem Ipsum text, so will need to update this. Plus validation works, but there are some issues with visibility of validation. I think this issue will go as another branch ;)

File: frontend/src/app/pages/login-page/login-page.component.html [hunk 1]
```diff
@@ -1,87 +1,106 @@
...
-        </mat-card-header>
-        <mat-card-content>
+<!--suppress XmlDuplicatedId -->
+<div class="row">
+    <header class="col-sm-12 col-lg-4">
+        <div class="logo">
+            <img src="../../../assets/icon/medtagger_logo.svg"/>
+            <h1>MedTagger</h1>
+        </div>
+        <h3>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam at leo vitae est
```
Comment (Darkoon) [inline frontend/src/app/pages/login-page/login-page.component.html]: Damn I really love Lorem ipsum but we need to change it to some MedTagger related text :)

File: frontend/src/app/pages/login-page/login-page.component.html [hunk 2]
```diff
@@ -1,87 +1,106 @@
...
+            <h4 [ngClass]="{active: loginPageMode == LoginPageMode.REGISTER}" (click)="changePageMode()">
+                <a routerLink="/login">Sign up</a>
+            </h4>
+            <h4 [ngClass]="{active: loginPageMode == LoginPageMode.LOG_IN}" (click)="changePageMode()">
+                <a routerLink="/login">Sign in</a>
+            </h4>
+        </div>
+        <main *ngIf="loginPageMode == LoginPageMode.LOG_IN">
+            <h3>Hello!</h3>
+            <h4>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam at leo vitae est fermentum suscipit quis
```
Comment (Darkoon) [inline frontend/src/app/pages/login-page/login-page.component.html]: Same as above, change lorem ipsum to project related text.

File: frontend/src/app/pages/login-page/login-page.component.html [hunk 3]
```diff
@@ -1,87 +1,106 @@
...
-                                      [diameter]="36"
-                                      strokeWidth="2"
-                                      *ngIf="registrationInProgress"></mat-progress-spinner>
-            </mat-card-actions>
-        </mat-card-content>
-    </mat-card>
-</div>
+                <mat-progress-spinner *ngIf="registrationInProgress" class="login-spinner" mode="indeterminate"
+                                      [diameter]="36" strokeWidth="2"></mat-progress-spinner>
+                <!--<mat-icon *ngIf="registrationInError" color="warn">error</mat-icon>-->
```
Comment (Darkoon) [inline frontend/src/app/pages/login-page/login-page.component.html]: What about that code?

File: frontend/src/styles.scss [hunk 4]
```diff
@@ -1,10 +1,34 @@
 /* You can add global styles to this file, and also import other style files */
+@import '~bootstrap/scss/bootstrap';
 @import './app-theme';
+@import "app/utils/mixins";
 
 body {
-    background: rgba(0, 0, 0, 0.1);
-    background-size: 30%;
-    margin: 0;
+    //background: rgba(0, 0, 0, 0.1);
```
Comment (Darkoon) [inline frontend/src/styles.scss]: Remove unused code.

File: frontend/src/app/pages/login-page/login-page.component.html [hunk 5]
```diff
@@ -1,13 +1,10 @@
...
 <div class="row">
     <header class="col-sm-12 col-lg-4">
         <div class="logo">
             <img src="../../../assets/icon/medtagger_logo.svg"/>
             <h1>MedTagger</h1>
         </div>
-        <h3>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam at leo vitae est
-            fermentum suscipit quis
-            vestibulum leo.</h3>
+        <h3>The first open-source platform for labeling and aggregation of datasets with a usage of a crowdsourcing idea.</h3>
```
Comment (jszemplinski) [inline frontend/src/app/pages/login-page/login-page.component.html]: `with usage`*

**Pull request merged**